remark-plugins/extract-image-class.js

```diff
@@ -0,0 +1,26 @@
+import { visit } from 'unist-util-visit'
+export function extractImageClass() {
+    return (tree) => {
+      visit(tree, "image", (node) => {
+        let [url, ending] = node.url.split("#");
```

Comment:
Making an assumption here that we have a compliant URL (that should not have multiple # signs)

Reply:
I'll do that and leave a note about the URL spec and multiple #s.